content/beginner/300_windows/cleanup.md

```diff
@@ -11,7 +11,7 @@ unalias calicoctl
 kubectl delete -f https://docs.projectcalico.org/archive/v3.15/manifests/calicoctl.yaml
 kubectl delete -f ~/environment/windows/sample-deployments.yaml
 kubectl delete -f ~/environment/windows/user-rolebinding.yaml
-kubectl delete -f https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/master/config/v1.6/calico.yaml
+kubectl delete -f https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/release-1.6/config/v1.6/calico.yaml
```

Comment:
Why you want to pin to this specific version ?

Reply:
The version is the same 1.6 version which is already used in this part of the workshop and in the [Network Policies workshop](https://www.eksworkshop.com/beginner/120_network-policies/calico/install_calico/)
I just corrected the link, because current link returns 404 now (/master/ should be changed to /release-1.6/ in the link.)
[Network Policies workshop](https://www.eksworkshop.com/beginner/120_network-policies/calico/install_calico/) already has the correct link.